tabs/window-control-position-macos.css

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*position window controls.  Only for MacOS*/
```

Comment:
Question: what does it do with the position of the window controls?

Reply:
It centers them in the navigation bar when there's no tab bar or title bar.

Reply:
I can't test this, but this seems weird and platform-inconsistent. Is there any reason for this behaviour? Or am I misunderstanding? Window controls are close/minimize/maximize, right? A screenshot might be welcome.

Reply:
With the style:
![screen shot 2017-09-03 at 9 58 22 am](https://user-images.githubusercontent.com/29472511/30004107-b8910580-908e-11e7-9ba3-a96e57deae07.png)

Without the style:
![screen shot 2017-09-03 at 9 59 42 am](https://user-images.githubusercontent.com/29472511/30004108-b89bfdd2-908e-11e7-8948-89fb3d1a3846.png)

It's not a major issue, but it puts the appearance more like other native MacOS apps like Safari.

Reply:
Ah, it *vertically* centers window controls! That explains a lot, lol. Thank you for your patient explanation :) I'll now work on rebasing this PR.

Reply:
Oh, one more thing: does this only apply when the tab bar is hidden? In this case, you can merge it with the other style.